### ISSUE TITLE
Feature Request: reStructuredText Format

### ISSUE THREAD

**Issue by hrittich** [asamuzaK/url2clipboard#43]
Hi,

thank you for your great add-on. I am often working with reStructuredText documents. In these documents, links are formated in the following way:

     `Link text <http://example.com/>`_

See, e.g., [here](http://www.sphinx-doc.org/en/stable/rest.html#hyperlinks). Could you add this format to the list of available export formats?

**Comment by asamuzaK** [comment]
Can you tell me the results of examples below?

```
`Link text containing ` backtick <http://example.com/>`_
`Link text containing \` escaped backtick <http://example.com/>`_
`Link text containing &#96; backtick entitiy ref <http://example.com/>`_
`Link text containing < lt sign <http://example.com/>`_
`Link text containing \< lt sign with escape <http://example.com/>`_
`Link text containing &#60; lt sign entity ref <http://example.com/>`_
```

**Comment by asamuzaK** [comment]
Found [Escaping Mechanism](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#id37 "Escaping Mechanism")

So, examples below should be the expected link text string?
```
`Link text containing \` escaped backtick <http://example.com/>`_
`Link text containing \< lt sign with escape <http://example.com/>`_
```

**Comment by hrittich** [comment]
I have tested all six example with docutils. The following seem to work as expected:

    `Link text containing ` backtick <http://example.com/>`_
    `Link text containing \` escaped backtick <http://example.com/>`_  
    `Link text containing < lt sign <http://example.com/>`_
    `Link text containing \< lt sign with escape <http://example.com/>`_

These two variants do _not_ work:

    `Link text containing &#96; backtick entitiy ref <http://example.com/>`_
    `Link text containing &#60; lt sign entity ref <http://example.com/>`_

I assume the escaped backtick and escaped less-than are the safest option.

**Comment by asamuzaK** [comment]
Thanks.
Should `*` be escaped too?
Or, should I leave it to the user to escape or not?

**Comment by hrittich** [comment]
I have tried the xpi linked above. Copying a reStructuredText in the right-click-menu works fine. However, I cannot click on the "reStructuredText" entry in the menu of the toolbar icon. Furthermore the entry is missing from the "Copy Link URL" and "Copy All Tabs URL".

**Comment by hrittich** [comment]
Works perfectly.

Thank you very much.

**Comment by asamuzaK** [comment]
Released v1.3.0 on AMO
